modules/ROOT/pages/build-app-practices.adoc

```diff
@@ -80,9 +80,11 @@ If you use Anypoint Connector for File (File Connector) and share part of the pa
 For HTTP and HTTPS, use the following placeholders, depending on the deployment model:
 
 * `${http.port}` +
+* `${https.port}` +
```

Comment:
per Facundo:

these *.port props are reserved in CH but not in on-prem Mule

hybrid is a standalone runtime, so you can define the value you want for those properties

it's a good practice to use those properties

Reply:
https://docs.mulesoft.com/runtime-manager/cloudhub-networking-guide

Reply:
https://docs.mulesoft.com/runtime-manager/cloudhub-manage-props#cloudhub-reserved-properties